saltychat/SaltyShared/Event.cs

```diff
@@ -16,6 +16,7 @@ public class Event
         public const string SaltyChat_TalkStateChanged = "SaltyChat_TalkStateChanged";
         public const string SaltyChat_MicStateChanged = "SaltyChat_MicStateChanged";
         public const string SaltyChat_SoundStateChanged = "SaltyChat_SoundStateChanged";
+        public const string SaltyChat_ConnectedToTeamspeak = "SaltyChat_ConnectedToTeamspeak"
```

Comment:
You missed a `;` and the event name is kind of weird, it would suggest that the WebSocket is connected to TeamSpeak, or the player is connected to the TeamSpeak server.
The `StateUpdate`-property is called `IsReady`, why don't you just use that?
```suggestion
        public const string SaltyChat_IsReady = "SaltyChat_IsReady";
```